index.html

```diff
@@ -50,85 +51,146 @@ <h1>Laura Jong</h1>
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder"
+          />
           <h3>Multi-Post Stories</h3>
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
-            has been the industry's standard dummy text ever since the 1500s, 
-            when an unknown printer took a standard dummy text.</p>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
           <ul class="container languages">
             <li>css</li>
             <li>html</li>
             <li>bootstrap</li>
             <li>Ruby</li>
           </ul>
-          <button>
-          See Project
-          </button>
+          <button>See Project</button>
         </article>
         <article class="container project">
-          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder"
+          />
           <h3>Multi-Post Stories</h3>
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
-            has been the industry's standard dummy text ever since the 1500s, 
-            when an unknown printer took a standard dummy text.</p>
+          <p> 
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
           <ul class="container languages">
             <li>css</li>
             <li>html</li>
             <li>bootstrap</li>
             <li>Ruby</li>
           </ul>
-          <button>
-          See Project
-          </button>
+          <button>See Project</button>
         </article>
       </section>
+      <section class="container contact">
+        <h2>CONTACT US</h2>
+        <p> 
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it 
+        </p>
+        <form action="https://formspree.io/f/mbjqaodr" method="POST">
+          <ul>
+            <li>
+              <input type="text" id="name" name="user_name" placeholder="Text Input" maxlength="30" required />
+            </li>
+            <li>
+              <input type="email" id="mail" name="user_email" placeholder="jeremy@gmail.com" required/>
+            </li>
+            <li>
+              <textarea name="user_message" id="msg" placeholder="Enter text here" maxlength="500" required></textarea>
+            </li>
+          </ul>
+          <button type="submit" value="Submit" class="submit">Submit</button>
```

Comment:
- [ ] [OPTIONAL] Let's center the submit button in accordance with the design. This might have been missed by mistake for I can see your work is meticulous, but we have to it. Consider making the `<form>` a flex container, then center its elements.

### REFERENCE

| Implementation | Figma Design |
| --- | --- |
| ![form_imp](https://user-images.githubusercontent.com/45002654/125569632-10f58af9-9974-4e45-8d61-bbea2a4f1d9b.PNG) | ![form_design](https://user-images.githubusercontent.com/45002654/125569586-7d069031-20da-4982-95ab-2cfb199a72fa.PNG) |